config/environments/development.rb

```diff
@@ -67,4 +67,15 @@
 
   # Uncomment if you wish to allow Action Cable access from any origin.
   # config.action_cable.disable_request_forgery_protection = true
+
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = {
+    address: 'smtp.gmail.com',
+    port: 587,
+    domain: 'gmail.com',
+    user_name: 'amanvhora745@gmail.com',
```

Comment:
try to use env variables for confidentials details like username, password, api keys etc.